### PR TITLE
celery worker log format following LOG_FORMAT env#9404

### LINKED ISSUE #9404: Celery worker log format not following "LOG_FORMAT" env
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.9.1

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

1. do some operations about knowledge-base, e.g., delete dataset
2. observe logs of dify-worker which don't follow "LOG_FORMAT" env

### ✔️ Expected Behavior

微信截图_20241016154817

These are logs of dify-app following “LOG_FORMAT” env.

### ❌ Actual Behavior

image

### PR DESCRIPTION
**Pull request by liuhaoran1212**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [ ] Please open an issue before creating a PR or link to an existing issue
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

Celery worker log are output to the "LOG_FILE" env using "LOG_FORMAT" env, and use the "LOG_TZ" in env.

Fixes #9404 

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [ ] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

**Comment by bowenliang123** [comment]
Consider merging #9129 before this PR, changing the timezone into `TimeZoneName ` type for easier use. cc @crazywoola @laipz8200

**Review by crazywoola — DISMISSED** [review]

**Comment by crazywoola** [comment]
Please fix the lint errors as well

File: api/pyproject.toml
Comment (bowenliang123) [inline api/pyproject.toml:121]: Is the worker logging a new feature in 5.4.x? Any doc reference for the reason  bumping celery version ?

Comment (liuhaoran1212) [reply api/pyproject.toml:121]: The worker_logfile is a new feature added in Celery 5.4, which allows the worker logs to be output to a specified log. Please see the attached file for details.

Comment (liuhaoran1212) [reply api/pyproject.toml:121]: [https://docs.celeryq.dev/en/stable/userguide/configuration.html](url)

**Review by bowenliang123 — APPROVED** [review]
LGTM.

**Comment by liuhaoran1212** [comment]
After the Code was committed, it seems that CI was not triggered, it only shows "2 workflows awaiting approval", Please tell me what might be the reason for this, thanks. @crazywoola @bowenliang123

**Comment by bowenliang123** [comment]
Because this is your first ever PR in this repo, all workflows are executed only when the maintainer approves the runs. You may need a hand from @laipz8200 @crazywoola . Or you may test it locally and raise your screenshot evidence for passing the tests.

**Comment by liuhaoran1212** [comment]
> Because this is your first ever PR in this repo, all workflows are executed only when the maintainer approves the runs. You may need a hand from @laipz8200 @crazywoola . Or you may test it locally and raise your screenshot evidence for passing the tests.

I understand now, thanks

**Comment by crazywoola** [comment]
Please rebase the main branch and fix the errors again. :)

**Review by crazywoola — DISMISSED** [review]

**Review by crazywoola — APPROVED** [review]

**Pull request merged**
